### DISCUSSION TITLE
How would I easily switch between using local source and the vcpkg source

### DISCUSSION THREAD

**Discussion by Astiolo** [microsoft/vcpkg#22149] [Q&A]
I'd like to use Vcpkg with internal CMake libraries, but they often need to be developed in tandem with the application. So I need a way that I can use Vcpkg to source the packages normally, and be able to easily switch to local source code for active development.

It looks like Conan handles this very nicely with "editable" mode https://docs.conan.io/en/latest/developing_packages/editable_packages.html. Does Vcpkg offer anything equivalent? or is there a nice CMake trick to get similar behavior?

**Comment by autoantwort** [comment]
AFAIK no
